python/lbann/modules/__init__.py

```diff
@@ -10,3 +10,7 @@
 from lbann.modules.rnn import LSTMCell, GRU, ChannelwiseGRU
 from lbann.modules.transformer import MultiheadAttention
 from lbann.modules.graph import *
+from lbann.modules.subgraph import *
+from lbann.modules.activations import *
+from lbann.modules.pytorch import *
+from lbann.modules.transformations import *
```

Comment:
I prefer being explicit about what we're importing:
```suggestion
from lbann.modules.activations import Gelu, Gelu_approx, Silu
from lbann.modules.pytorch import PytorchMatmul, PytorchLinear, PytorchLayerNorm
from lbann.modules.transformations import Permute, Cumsum
```
We ran into some trouble from blindly importing `lbann.modules.subgraph` (https://github.com/LLNL/lbann/pull/2026).